README.md

```diff
@@ -1,17 +1,101 @@
-# dashd-prometheus-exporter
-![dashd prometheus exporter logo](dashd-prometheus-exporter.png)
+# bitcoind-prometheus-exporter
+Bitcoind wallet metrics **Prometheus** exporter.
 
-Dash headless wallet daemon exporter for Prometheus.
+> `bitcoind-prometheus-exporter` is compatible with most **bitcoin** forks.
+
+Produce **wallet** metrics, own **addresses** but also **blockchain** infos.
+Most important metrics are:
+* wallet total balance
+* wallet version
+* if the wallet is unlocked
+* available (spendable) balance for each managed addresses (and watched addresses)
+* best block time and index
+* ...
 
 ## Usage
+Edit the `.env` environment file to suit your need and run:
+```
+npm start
 ```
-docker run --name my-exporter -p 3000:3000 -e "rpcuser=myrpcuser" -e "rpcpassword=myrpcpassword" -e "rpchost=my-wallet" --link my-wallet lepetitbloc/dashd-prometheus-exporter
+> `bitcoind-prometheus-exporter` uses the `bitcoind` **JSON-RPC** API under the hood and need those credentials:
+> ```
+>rpcuser=test
+>rpcpassword=1cf98b57-5i09-4fa1-9c07-2e28cb2cb47b
+>```
+
+## Usage with other wallets
+The following environment variables are available, just change them to suit your needs:
+> ```
+>symbol=DASH
```

Comment:
`symbol=BTC` ?

Reply:
This is under the *Usage with other wallets* section, that's why it is not BTC here, but maybe this is confusing?